sql/o.karmanskyi_homework_27.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE dbo.LibraryUser
+(
+	Id INT IDENTITY(1, 1) PRIMARY KEY NOT NULL,
+	FirstName VARCHAR(50) NOT NULL,
+	LastName VARCHAR(50) NOT NULL,
+	DateRegistered DATETIME NOT NULL,
+)
+
+CREATE TABLE dbo.BookAuthor
```

Comment:
not critical, but in future your library can have magazines as well and it won't be good to reference for magazine book authors :)
simplify name just to Authors